biztalk/core/sftp-adapter.md

```diff
@@ -45,14 +48,15 @@ BizTalk Server 2013 and BizTalk Server 2013 R2 use older ssh library instead of
     |Log| Available starting with BizTalk Server 2016. <br/><br/>Enter the full path to create a client-side log file. Use this log file to troubleshoot any errors.|
-    |Maximum Connection Reuse Time In Seconds| Available starting with BizTalk Server 2016 CU 7. <br/><br/>The maximum connection reuse time allows connections to be gracefully closed and removed from the pool after a connection has been in use for a specific amount of time. A value is 0 or less indicates that this behaviour is disabled.|
+    |Maximum Connection Reuse Time In Seconds| Available starting with BizTalk Server 2016 CU 7. <br/><br/>The maximum connection reuse time allows connections to be gracefully closed and removed from the pool after a connection has been in use for a specific amount of time. It is recommended to set a value here, e g 240 seconds, to release temporary session files. A value is 0 or less indicates that this behaviour is disabled.|
+    |Transfer Mode| Available starting with BizTalk Server 2020 CU 3. <br/><br/>The WinSCP transfer mode. Default is binary. ASCII and Automatic can also be selected. Binary default mode is normally good for most scenarios, but in certain UNIX, mainframe text file transfer scenarios, ASCII mode may be needed for proper line end conversion.|
 
      **Polling**
 
     |Use this|To do this|
     |--------------|----------------|
     |Enable Timestamp Comparison|Available starting with BizTalk Server 2016 cumulative update 6. <br/><br/>If **Retain After Download** is set to True, this property determines whether a change in file timestamp will trigger a re-download of the file.<br /><br /> **Default value:** False|
-    |Polling Interval|Specify the intervals at which the adapter polls the server. To poll continuously, set this value to zero.<br /><br /> **Default value:** 5|
+    |Polling Interval|Specify the intervals at which the adapter polls the server. To poll continuously, set this value to zero. If you set this to a low value you may get high CPU consumption. It is recommended to set this to as high value as possible. You should also consider that the DIR listing can take long time if there are lots of files in the folder you poll. <br /><br /> **Default value:** 5|
```

Comment:
Again, not sure if this belongs in mainline docs. Maybe we could have a section for common problems and suggested solutions.

Reply:
This is also one of our common customer cases we get very often where customers get into due to our default settings being sub-optimal. Sometimes with 1000 files in the folder and slow connection on busy SFTP server, only the DIR listing can take 20 seconds and when polling default value is 5 seconds, it is causing constant high CPU polling. I think it is good to have it visible where customers will see it. But we can move down also to later FAQ section, but afraid many will miss this. A good default value for SFTP receive polling in my opinion would be more 120 seconds to avoid high cpu on BizTalk and high load on SFTP servers.